AtlasLootClassic/Data/VendorPrice.lua

```diff
@@ -1287,7 +1303,7 @@ local VENDOR_PRICES = {
     [15198] = "pvpAlterac:20:pvpArathi:20:pvpWarsong:20",
     [15197] = "pvpArathi:3:pvpWarsong:3",
     [15196] = "pvpArathi:3:pvpWarsong:3",
-    [12584] = "honor:19125:pvpArathi:20",
+    [12584] = "honor:9563:pvpArathi:20",
```

Comment:
[32807] = "honor:17850:pvpArathi:40",
    [32799] = "honor:17850:pvpArathi:40",
    [32979] = "honor:17850:pvpArathi:40",
    [32798] = "honor:17850:pvpArathi:40",
    [32800] = "honor:17850:pvpArathi:40",
    [32802] = "honor:17850:pvpArathi:40",
    [32808] = "honor:17850:pvpArathi:40",
    [32998] = "honor:17850:pvpArathi:40",
    [32803] = "honor:17850:pvpArathi:40",
    [32804] = "honor:17850:pvpArathi:40",
    [32797] = "honor:17850:pvpArathi:40",
    [32806] = "honor:17850:pvpArathi:40",
    [32988] = "honor:17850:pvpArathi:40",
    [32805] = "honor:17850:pvpArathi:40",
    [32801] = "honor:17850:pvpArathi:40",
    [32795] = "honor:17850:pvpEye:40",
    [32787] = "honor:17850:pvpEye:40",
    [32981] = "honor:17850:pvpEye:40",
    [32786] = "honor:17850:pvpEye:40",
    [32788] = "honor:17850:pvpEye:40",
    [32790] = "honor:17850:pvpEye:40",
    [32796] = "honor:17850:pvpEye:40",
    [32999] = "honor:17850:pvpEye:40",
    [32791] = "honor:17850:pvpEye:40",
    [32792] = "honor:17850:pvpEye:40",
    [32785] = "honor:17850:pvpEye:40",
    [32794] = "honor:17850:pvpEye:40",
    [32990] = "honor:17850:pvpEye:40",
    [32793] = "honor:17850:pvpEye:40",
    [32789] = "honor:17850:pvpEye:40",
    [32820] = "honor:11794:pvpWarsong:20",
    [32811] = "honor:11794:pvpWarsong:20",
    [32980] = "honor:11794:pvpWarsong:20",
    [32810] = "honor:11794:pvpWarsong:20",
    [32814] = "honor:11794:pvpWarsong:20",
    [32812] = "honor:11794:pvpWarsong:20",
    [32821] = "honor:11794:pvpWarsong:20",
    [32997] = "honor:11794:pvpWarsong:20",
    [32816] = "honor:11794:pvpWarsong:20",
    [32817] = "honor:11794:pvpWarsong:20",
    [32809] = "honor:11794:pvpWarsong:20",
    [32819] = "honor:11794:pvpWarsong:20",
    [32989] = "honor:11794:pvpWarsong:20",
    [32818] = "honor:11794:pvpWarsong:20",
    [32813] = "honor:11794:pvpWarsong:20",
    [31984] = "arena:3750",
    [31966] = "arena:3750",
    [32025] = "arena:3750",
    [32055] = "arena:3750",
    [31959] = "arena:3750",
    [32014] = "arena:3750",
    [31986] = "arena:3750",
    [32044] = "arena:2625",
    [32026] = "arena:2625",
    [32052] = "arena:2625",
    [31965] = "arena:2625",
    [32028] = "arena:2625",
    [32053] = "arena:3150",
    [32963] = "arena:3150",
    [32964] = "arena:3150",
    [32046] = "arena:1125",
    [31958] = "arena:1125",
    [32027] = "arena:1125",
    [31985] = "arena:1125",
    [32003] = "arena:1125",
    [31978] = "arena:1125",
    [32961] = "arena:1125",
    [32045] = "arena:1875",
    [33309] = "arena:1875",
    [33313] = "arena:1875",
    [32962] = "arena:1000",
    [32054] = "arena:1000",
    [33943] = "arena:1000",
    [33946] = "arena:1000",
    [33076] = "arena:1000",
    [33077] = "arena:1000",
    [33937] = "arena:1000",
    [33949] = "arena:1000",
    [33078] = "arena:1000",
    [33952] = "arena:1000",
    [33940] = "arena:1000",

Reply:
Rest of s2 items, not including sets.

Reply:
Done👍